packages/webdriverjs/test/fixtures/iframe-lazy-1.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```

Comment:
NBD - Do we want these fixtures to live in the axe-fixtures repo?

Reply:
Good question. I should probably move them since puppeteer also has this same issue. I'll add this as a tech debt ticket since I'd like to get this released without waiting to add it to the fixture repo.

Reply:
https://github.com/dequelabs/axe-test-fixtures/issues/22